source/includes/_feature-importing.md

```diff
@@ -128,6 +128,139 @@ object is explicitly unordered. If you wish the variables to have an order,
 you must supply an order object rather than relying on any order of the
 "metadata" object.
 
+
+It is possible to create arrays and derived arrays simulaneously in one request
+when creating the dataset along its metadata. The variable references inside the
+derivation expressions must point to declared aliases of variables or 
+subvariables. 
+
+```http
+POST /datasets/ HTTP/1.1
+Content-Type: application/shoji
+Content-Length: 3294
+...
+{
+    "element": "shoji:entity",
+    "body": {
+      "name": "Dataset with derived arrays",
+      "table": {
+        "metadata": {
+          "CA3": {
+            "name": "cat array 3", 
+            "derivation": {
+              "function": "array", 
```

Comment:
The references aren't optional. Else you get a virtual variable without name/alias and shit breaks.